kube-runtime/src/lib.rs

```diff
@@ -22,7 +22,6 @@ k8s_openapi::k8s_if_ge_1_19! {
     pub mod events;
 }
 pub mod finalizer;
-mod observer;
```

Comment:
I still think the traitification here makes sense eventually, but I don't think the backoff PR needs to block on that.